DirectX11Proj/ResourceManager.h

```diff
@@ -35,7 +36,10 @@ class ResourceManager
 	std::unique_ptr<ModelLoader> mpModelLoader;
 	ID3D11Device* mpDevice;
 
-	std::vector<std::shared_ptr<ModelClass>> mLoadedModels;
+	// Turn these into maps at one point
+	std::vector<std::unique_ptr<ModelClass>> mLoadedModels;
+	std::vector<std::unique_ptr<d3dTexture>> mLoadedTextures;
```

Comment:
Just want to say that you did the right thing by not naming it mpLoadedModels, as the variable isn't a pointer itself. Gj 👍